other/materials_designer/create_custom_shape.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>Let's add an image of the final result to showcase the final result</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>